continuous-upgrade/jobs/provision-job.yml

```diff
@@ -0,0 +1,12 @@
+- job:
+    name: continuos-upgrade/install-job
+    project-type: freestyle
+    auth-token: origin1
+    defaults: global
+    builders:
+      - shell: 
+          !include-raw:
+              - install-oct.sh
+              - terminate-lib.sh
```

Comment:
I am not sure this is right -- we should only terminate on user ask. If something goes wrong we want it to stick around until someone comes to clean it up.